usr/share/rear/output/USB/Linux-i386/850_make_USB_bootable.sh

```diff
@@ -59,10 +59,11 @@ case "$usb_filesystem" in
         fi
         ;;
     ("")
-        BugError "Filesystem where the booting related files are on $RAW_USB_DEVICE could not be found"
+        LogPrintError "Could not find a filesystem for $BUILD_DIR/outputfs in /proc/mounts"
+        Error "An ext2/3/4 or vfat filesystem must be mounted for the booting related files on $RAW_USB_DEVICE"
```

Comment:
I would say `mounted on $BUILD_DIR/outputfs` to make even clearer what the expectation is.

Reply:
My idea was to show both $BUILD_DIR/outputfs and $RAW_USB_DEVICE
because $BUILD_DIR/outputfs is a meaningless directory like
```
/var/tmp/rear.Bhlh9JjUxW91R0i/outputfs
```
so $RAW_USB_DEVICE provides some context to the user
that this two messages are about his USB device.

Reply:
Fixed (hopefully sufficiently for this pull request) via
https://github.com/rear/rear/commit/5d86ec547e03fc8fa7d635b46f642682662004da